robot/lib/ipmi_client.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Documentation           This module is for executing ipmitool commands.
+
+Resource                ../lib/resource.txt
+
+Library                 SSHLibrary
+Library                 OperatingSystem
+
+*** Variables ***
+
+${IPMI_CMD}       ipmitool -H ${OPENPOWER_HOST} -I lanplus -U ${OPENPOWER_USERNAME} -P ${OPENPOWER_PASSWORD}
```

Comment:
You don't have to have a user name or password, or can only have one. So this will be inadequate to test existing systems.